pages/getting_started/create_dashboard.rst

```diff
@@ -1,36 +1,41 @@
 Create Your Dashboard
 ---------------------
 
-You should be at a screen like the one below. If you dozed off or went to cook some meatballs, go to System -> Inputs, select the Syslog UDP input you created, and hit Show messages.
-
-Now it’s go-time.
-
 You’ve got data coming in, let’s add information to a dashboard to better visualize the data we want to see.
 
 Add a Dashboard
-^^^^^^^^^^^^^^^^^^
+^^^^^^^^^^^^^^^
 
-We'll start by adding the message count data to a dashboard. Click *Add count to dashboard*, and it will say *No Dashboards, create one?*   Yes!  Click that.
+Now we'll create a new dashboard for these messages by opening the *Dashboards* page in the top menu and clicking on *Create dashboard*.
 
-.. image:: /images/gs_11-createdash.png
+.. image:: /images/gs/graylog_dashboard.png
 
 Give your new dashboard a title and description.
 
 .. image:: /images/gs_12-titledash.png
 
+In this given example the title is *Important Stuff* and the description *Look at this, yo.*
+
 Add a Dashboard Widget
 ^^^^^^^^^^^^^^^^^^^^^^
 
+Click the "Show received messages" button next to the *appliance-syslog-udp* input on the System / Inputs page to only show messages received by this input.
```

Comment:
The reference to the inputs page should be in the same style as the other occurrences:
```
*System > Inputs*
```